hacs.json

```
@@ -1,5 +1,7 @@
 {
   "name": "ICY E-thermostaat",
+  "zip_release": true,
+  "filename": "combined.zip",
```

```suggestion
  "filename": "combined.zip",
  "hide_default_branch": true,
```

This will hide the option to install from "master" and get the "0.0.0" version implemented in #12